test/Twig/Tests/ErrorTest.php

```diff
@@ -207,6 +207,20 @@ public function getErroredTemplates()
             ],
         ];
     }
+
+    public function testTwigLeakOutputInDebugMode()
+    {
+        $output = exec(sprintf('%s %s debug', \PHP_BINARY, __DIR__.'/Fixtures/errors/leak-output.php'));
```

Comment:
you need to escape the argument for the path (`__DIR__` might have spaces in it)

Reply:
Good catch. https://github.com/twigphp/Twig/pull/3062